cmd/ctr/commands/images/tag.go

```diff
@@ -60,6 +65,9 @@ var tagCommand = cli.Command{
 
 		if !context.BoolT("local") {
 			for _, targetRef := range context.Args()[1:] {
+				if _, err := reference.ParseAnyReference(targetRef); !context.Bool("skip-reference-check") && err != nil {
```

Comment:
@dmcgowan Added a new flag, to skip the strict reference name check. 

But IMHO, doesn't it make the code less clean?

Also, should we have this as a global flag, so that other commands can also have the flag to skip the strict checks